tests/Oryx.Integration.Tests/Python/Python27EndToEndTests.cs

```diff
@@ -65,8 +65,10 @@ public async Task CanBuildAndRunPythonApp_UsingPython27_AndExplicitOutputStartup
                 });
         }
 
-        [Fact]
-        public async Task CanBuildAndRun_Python27App_UsingVirtualEnv()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
```

Comment:
i would keep one more test (if it  is not already there) that is without --compressDestinationDir

Reply:
If you notice the test, the `false` variation here tests without the `--compress-destination-dir`